src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/AdvSimd.PlatformNotSupported.cs

```diff
@@ -10,7 +10,12 @@ namespace System.Runtime.Intrinsics.Arm
     /// This class provides access to the ARM AdvSIMD hardware instructions via intrinsics
     /// </summary>
     [CLSCompliant(false)]
-    public abstract class AdvSimd : ArmBase
+#if INTERNAL_NULLABLE_ATTRIBUTES
```

Comment:
I think this needs to be a different ifdef 😄

Reply:
Maybe `INTERNAL_HWINTRINSICS`?

Reply:
Yup yup, this was me testing a hack to see if CI complains :D (though I guess I can test the full fix itself right now)